docker/nginx/server.conf

```diff
@@ -0,0 +1,37 @@
+server {
+    listen 80;
+
+    server_name localhost;
+    root /var/www/web;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
+    }
+
+    # DEV
+    location ~ ^/(app_dev|config)\.php(/|$) {
```

Comment:
Dropping app_dev.php (as asked in a previous comment) impact this. (this section should probably simply dropped)